pom.xml

```diff
@@ -14,7 +14,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.447</version>
+        <version>1.567</version>
```

Comment:
https://wiki.jenkins-ci.org/display/JENKINS/Choosing+Jenkins+version+to+build+against